extractor/src/main/java/org/schabi/newpipe/extractor/services/bandcamp/BandcampService.java

```diff
@@ -0,0 +1,126 @@
+// Created by Fynn Godau 2019, licensed GNU GPL version 3 or later
```

Comment:
@TobiGr @Stypox Do we really want this license line in every file?

Reply:
I don't know. I think they should either be in every file of the project or in none. I'd go with the latter since I don't see how they are useful and they are just a burden to maintain up-to-date.

Reply:
When code is not licensed, you are by default not allowed to use it. Therefore, the GNU GPL recommends to explicitly grant the license at the beginning of each file, making it extra clear that you are allowed to use it. Here I kept it extra short so that it only takes one line.

Reply:
I'd also keep the license.